packages/smart-account/src/SmartAccount.ts

```diff
@@ -411,8 +411,8 @@ class SmartAccount extends EventEmitter {
       sender: this.address,
       target: this.address,
       nonce: gasTankNonce,
-      callData: execTransaction.data || '',
-      callGasLimit: BigNumber.from(500000), // will be updated below
+      callData: execTransactionData || '',
```

Comment:
what is the difference here execTransactionData and execTransaction.data?
how it was working earlier as I don't see other change

Reply:
I see